main.go

```diff
@@ -420,6 +420,12 @@ func doMain(args []string) error {
 
 	env := NewEnvironment(new(osCalls))
 	device := NewDevice(env, new(osCalls), config.GetDeviceConfig())
+	ap, err := device.GetActive()
+	if err != nil {
+		log.Error("Failed to read the current active partition.")
+	} else {
+		log.Infof("Mender running image: %s", ap)
```

Comment:
I would add somewhere the world partition `Running the Mender image from partition: %s`